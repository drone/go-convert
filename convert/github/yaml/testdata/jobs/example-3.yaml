# https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-setting-permissions-for-a-specific-job

jobs:
  call-workflow:
    uses: octo-org/example-repo/.github/workflows/called-workflow.yml@main
    with:
      username: mona
