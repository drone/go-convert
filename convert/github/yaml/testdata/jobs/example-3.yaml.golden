jobs:
  call-workflow:
    uses: octo-org/example-repo/.github/workflows/called-workflow.yml@main
    with:
      username: mona